src/ceph_osd.cc

```diff
@@ -56,6 +56,10 @@ TracepointProvider::Traits osd_tracepoint_traits("libosd_tp.so",
                                                  "osd_tracing");
 TracepointProvider::Traits os_tracepoint_traits("libos_tp.so",
                                                 "osd_objectstore_tracing");
+#ifdef WITH_OSD_INSTRUMENT_FUNCTIONS
+TracepointProvider::Traits cyg_profile_traits("libcyg_profile_tp.so",
```

Comment:
@tchaikov GCC names the added functions "__cyg_profile_func_enter" and "__cyg_profile_func_exit". By looking online, it seems to be because it was added by Cygnus Solutions into GCC. So pretty much an arbitrary name. :)